cuqi/experimental/mcmc/_direct.py

```diff
@@ -0,0 +1,18 @@
+from cuqi.experimental.mcmc import SamplerNew
+
+class Direct(SamplerNew):
```

Comment:
Should we call this `DirectNew` to be consistent with naming, even if we didn't have a `Direct` before (other than the `Direct` defined in the Gibbs notebook, so `DirectNew` would clearly distinguish it from that one)

Reply:
Agreed. Done.